DDD.Functions/local.settings.json

```diff
@@ -51,19 +59,25 @@
     "WaitingListCanVoteWithEmail": "true",
     "MinVotes": "6",
     "MaxVotes": "12",
+
     // EloVoting
-    "EloVotesConnectionString": "UseDevelopmentStorage=true",
+    "EloVotesConnectionString": "DefaultEndpointsProtocol=https;AccountName=dddperthdatatest;AccountKey=gxUQ5OeKRqmmS4qdz0w1xsPaA09fX+RrkzkCK4LAodq//kwJm0kj8Y9FGbyimtrVZahrQq7MmubncW+f68+FCg==;EndpointSuffix=core.windows.net",
     "EloVotingTable": "EloVotes",
     "EloPasswordPhrase":"Perth2022",
     "EloEnabled": "true",
     "EloAllowedTimeInSecondsToSubmit": "120",
+    "UserVotingSessionsConnectionString": "AccountEndpoint=https://dddperthdatatest.documents.azure.com:443/;AccountKey=UbR2GBvA5PPok2ctY7t5aXetTCDPokB9Z4BxJoeKj51JwfKsr97ql2022IUawL12QJAx9VJmM2nVq7jq94yeTg==;",
+    "UserVotingSessionsDatabaseId": "dddperthdatatest",
+    "UserVotingSessionsContainerId": "votesessions",
```

Comment:
Is it container or db table? A bit confusing for new people

Reply:
These are the names that are used by CosmosDb, I agree they are confusing tho :-)